pkg/controllers/managementuser/rbac/globalrolebinding_handler.go

```diff
@@ -79,59 +75,48 @@ func newGlobalRoleBindingHandler(workload *config.UserContext) v3.GlobalRoleBind
 // grbHandler ensures the global admins have full access to every cluster. If a globalRoleBinding is created that uses
 // the admin role, then the user in that binding gets a clusterRoleBinding in every user cluster to the cluster-admin role
 type grbHandler struct {
-	clusterName                 string
-	clusterRoleBindings         rbacv1.ClusterRoleBindingInterface
-	crbLister                   rbacv1.ClusterRoleBindingLister
-	grbIndexer                  cache.Indexer
-	grLister                    v3.GlobalRoleLister
-	rbLister                    rbacv1.RoleBindingLister
-	roleBindings                rbacv1.RoleBindingInterface
-	globalroleBindingController v3.GlobalRoleBindingController
-	clusters                    v3.ClusterInterface
-	provClusters                provisioningcontrollers.ClusterCache
+	clusterName         string
+	clusterRoleBindings rbacv1.ClusterRoleBindingInterface
+	crbLister           rbacv1.ClusterRoleBindingLister
+	grbCache            mgmtv3.GlobalRoleBindingCache
+	crtbCache           mgmtv3.ClusterRoleTemplateBindingCache
+	crtbCtrl            mgmtv3.ClusterRoleTemplateBindingClient
+	grLister            v3.GlobalRoleLister
```

Comment:
Removed unused fields and used Wrangler for the newly added controllers and caches